app/src/main/java/ch/epfl/sdp/drone3d/service/storage/data/MappingMission.kt

```diff
@@ -4,7 +4,6 @@
  */
 
 package ch.epfl.sdp.drone3d.service.storage.data
-
```

Comment:
A space between package and import should be kept

Reply:
Done